### PR TITLE
feat: Avatar Component Added

### PR DESCRIPTION
**Pull request by krate27**
Preview: 

Screenshot (508)

Screenshot (509)

Screenshot (510)

### CODE REVIEW

File: Components/Avatar.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+.bg-color{
+background-color: var(--Primary-color);
+color: var(--white-color);
+}
+.round-img{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.Avatar-wrapper{
```
Comment (bhanuprakashj) [inline Components/Avatar.css:31]: here class naming convention is bit different compared to other classes

File: Components/Avatar.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    width: 80px;
+    height: 80px;
+}
+.avatar-size-sm{
+    width: 60px;
+    height: 60px;
+}
+.avatar-size-xs{
+    width: 40px;
+    height: 40px;
```
Comment (Hamdrive) [inline Components/Avatar.css:20]: Consider using `rems` instead, if you have defined any root variables

File: Components/Avatar.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+}
+.Avatar-wrapper{
+    display: flex;
+    flex-direction:column;
+    gap: 20px;
+    max-width: 1000px;
+}
+.display-com{
+    display: flex;
+    border: 3px solid lightgray;
```
Comment (Hamdrive) [inline Components/Avatar.css:39]: try to use locally defined colors, as this can change from browser to browser

File: Components/Villainess.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    --warning-color: #ffff80;
+    --primary-color-toast: #33adff;
+    --secondary-color: #ccccb3;
+    --btn-color: #2B3784;
+    --white-color: #fff;
+    --Primary-color: #CC0033;
+    --body-bg-color: #e0e0eb;
+    --btn-hover-color: #990033;
+    --No-style-btn-color-card: #660033;
+    --No-style-btn-bg-color-card: #b1131375;
```
Comment (Hamdrive) [inline Components/Villainess.css]: Should be lowercase

File: Components/Villainess.css [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+}
+::-webkit-scrollbar {
+  width: 6px; /* width of the entire scrollbar */
+}
+::-webkit-scrollbar-track {
+  /* background: rgb(6, 7, 6);  */
+  background: var(--white-color); 
+}
+::-webkit-scrollbar-thumb {
+   /* background-color: blue;  */
```
Comment (Hamdrive) [inline Components/Villainess.css]: Cleanup unused commented code

File: Docsite-HTML-Files/Avatar.html [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+    ></script>
+    <link rel="stylesheet" href="/Components/Villainess.css" />
+    <link rel="stylesheet" href="" />
+  </head>
+  <body>
+    <!-- Navbar -->
+
+    <nav class="land-navbar">
+      <h1 class="land-heading-title">
+        V / N <br />
```
Comment (bhanuprakashj) [inline Docsite-HTML-Files/Avatar.html:19]: better not to use break tags

File: Docsite-HTML-Files/Avatar.html [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Avatar Component</title>
+    <script
+      src="https://kit.fontawesome.com/c9ffd32afb.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="/Components/Villainess.css" />
+    <link rel="stylesheet" href="" />
```
Comment (Hamdrive) [inline Docsite-HTML-Files/Avatar.html]: Empty link tag?

File: Docsite-HTML-Files/Avatar.html [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+      <p> Avatars are images that users can set as their profile picture. They can be custom photos, uploaded by users, or generated as Identicons as a placeholder.</p>
+      <p> <strong>Villainess</strong> Provides two kinds of Avatars , Round and Text respectively. </p>
+
+    <h1> Round Image Avatar </h1>
+    <p> There are images in four different sizes, to use them you need to include a common class <strong> round-img </strong> and use <strong>avatar-size-lg , avatar-size-md, avatar-size-sm, avatar-size-xs</strong> respectively as displayed below. </p>
+
+      <div class="display-com">
+      <img
+        src="/images/Avatar-img.jpg"
+        alt=""
```
Comment (Hamdrive) [inline Docsite-HTML-Files/Avatar.html]: Do write some message in `alt` attribute, it's good practice

File: Landing-Page-CSS/nav.css [hunk 9]
```diff
@@ -0,0 +1,113 @@
+.land-navbar {
+    display: flex;
+    /* justify-content: space-between; */
```
Comment (Hamdrive) [inline Landing-Page-CSS/nav.css]: Cleanup comments

File: Landing-Page-CSS/nav.css [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+    margin: 1rem;
+}
+
+.land-navbar-links .landing-page-link:hover {
+    color: var(--primary-text-color);
+}
+
+.land-hamburgerBtn {
+    position: absolute;
+    top: .75rem;
```
Comment (Hamdrive) [inline Landing-Page-CSS/nav.css:36]: For small values use `px` instead

File: Docsite-HTML-Files/Avatar.html [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+    </main>
+  </div>
+
+  <div class="footer-basic">
+    <footer>
+        <p class="signature"> Made with <i class="fas fa-code"></i> by Rohan Dubey </p>
+        <div class="social"><a href="/"> <i class="fab fa-github fa-lg"></i> </a>
+            <a href="/"> <i class="fab fa-twitter fa-lg"></i> </a>
+            <a href="/"> <i class="fab fa-linkedin fa-lg"></i> </a>
+            <p class="copyright">Villainess © 2022 </p>
```
Comment (bhanuprakashj) [inline Docsite-HTML-Files/Avatar.html:136]: using html entity(&#169;) for copyright is better i feel

File: Docsite-HTML-Files/Avatar.html [hunk 12]
```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Avatar Component</title>
+    <script
```
Comment (Hamdrive) [inline Docsite-HTML-Files/Avatar.html:8]: you can instead use the FontAwesome CDN and import it directly into your CSS file
Like you have done for Google Fonts

**Review by ivinayakg — APPROVED** [review]
looks clean to me, good work.

**Review by bhanuprakashj — COMMENTED** [review]
rest all looks good to me

**Review by Hamdrive — CHANGES_REQUESTED** [review]
overall good work done, improvements can be made. Have suggested some changes which should be considered.

File: Docsite-HTML-Files/Avatar.html (hunk 6 above)
Comment (krate27) [reply Docsite-HTML-Files/Avatar.html:19]: any specific reason?

**Review by Hamdrive — APPROVED** [review]
Looks good now, all changes requested and implemented

**Pull request closed without merging**